tests/unit/retrievers/test_text2cypher.py

```diff
@@ -310,3 +311,32 @@ def test_t2c_retriever_with_custom_prompt_bad_prompt_params(
     llm.invoke.assert_called_once_with(
         """This is a custom prompt. test ['example A', 'example B']"""
     )
+
+
+@patch("neo4j_graphrag.retrievers.base.Retriever._verify_version")
+@patch("neo4j_graphrag.retrievers.text2cypher.get_schema")
```

Comment:
Does the order matter here? Why are the orders for the other tests changed?

Reply:
Yes, the order matters, when using a decorator, the mocks are introduced in a given order: see [here](https://docs.python.org/3/library/unittest.mock.html#quick-guide) (you need to scroll down a bit to `def test(MockClass1, MockClass2):` and read note below)

Reply:
Though I need to check to make sure my update is using the proper one (I'm always getting confused about which one comes first)

Reply:
I checked again, and I think it's alright now.